src/Microsoft.PowerShell.Commands.Utility/commands/utility/CSVCommands.cs

```diff
@@ -1561,24 +1555,23 @@ private string
         }
 
         /// <summary>
-        /// This function reads the characters till next delimiter and adds them
-        /// to current
+        /// This function reads the characters till next delimiter and adds them to current.
         /// </summary>
         /// <param name="current"></param>
         /// <param name="endOfRecord">
-        /// this is true if end of record is reached
-        /// when delimiter is hit. This would be true if delimiter is NewLine
+        /// This is true if end of record is reached
```

Comment:
I have not set a goal to fix all the problems since there are too many. The main goal is to remove as many problems as possible to kontribjutery not diverted on these messages.

In this case, the IDE performs the formatting themselves when they show this pop-up comment. So deleting strings is better. On the other hand we need to see them on a screen width of 80-120 characters.  Although I could miss something-I did not try to catch all the cases.